cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -1802,69 +1891,67 @@ features:
       last30Days: Last 30 days
       sinceInception: Since Inception
     statsContainer:
-      avgRunTime: Average Duration
-      totalRuns: Total Runs
+      avgRunTime: Average duration
+      totalRuns: Total runs
     title: 'Summary'
 
   PipelineTriggers:
-    collapsedTabLabel: "Show Inbound Triggers ({count})"
+    collapsedTabLabel: "Show inbound triggers ({count})"
     description: Description
     EnabledTriggers:
       buttonLabel: Disable Trigger
       pipelineCount:
         0: "No pipelines set as trigger"
         1: "1 pipeline is set as trigger"
-        _: "{context.count} pipelines is set as trigger"
-      tabLabel: "View Enabled Triggers ({count})"
+      tabLabel: "View enabled triggers ({count})"
       title: "View pipelines enabled to trigger pipeline \"{pipelineName}\""
     Events:
       COMPLETED: Succeeds
       FAILED: Fails
       KILLED: Stops
-    expandedTabLabel: "Hide Inbound Triggers ({count})"
+    expandedTabLabel: "Hide inbound triggers ({count})"
     helperText: "\"{pipelineName}\" is triggered when this pipeline"
     namespace: Namespace
     pipelineName: Pipeline Name
     ScheduleRuntimeArgs:
       configure_enable_btn: Configure and Enable Trigger
       DefaultMessages:
-        choose_runtime_arg: Pick Runtime Argument
-        choose_plugin: Pick Plugin
-        choose_plugin_property: Pick Plugin Property
-        choose_runtime_arg_map: Pick Runtime Argument
```

Comment:
not needed?

Reply:
`choose_runtime_arg_map` is not needed, since it's the same as `choose_runtime_arg`. The rest I just lowercased